### ISSUE TITLE
Review-2018.09.23-李明鸣

### ISSUE THREAD

**Issue by lmmingzzz** [DIYer22/weekly-review#292]
### 本周工作
1.改D25论文3.3节，调整了3.1节和3.2节的结构
2.看了几篇早期的论文。
《improving deep neural networks for LVCSR using rectified linear units and dropout》
《Dropout: A Simple Way to Prevent Neural Networks from overfitting》
《Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift》《ImageNet Classification with Deep Convolutional Neural Networks》
发现batch norm这篇论文里，虽然BN算法就那么一点，但是论文里还是有一些不明白的地方。像第二章Towards Reducing Internal Covariate Shift看的就不是很明白。

### 下周工作
1.接着改D25论文。
2.接着看论文。
3.数字图像处理感觉要先放一放。

**Comment by ilydouble** [comment]
推荐焦李诚的那书，虽然不好看懂
